components/server/src/gitlab/gitlab-context-parser.spec.ts

```diff
@@ -531,19 +531,16 @@ class TestGitlabContextParser {
     @test public async testEmptyProject() {
         const result = await this.parser.handle({}, this.user, 'https://gitlab.com/gp-test-group/gp-test-empty-project');
         expect(result).to.deep.include({
-            "ref": undefined,
-            "refType": undefined,
-            "revision": undefined,
             "isFile": false,
             "repository": {
                 "host": "gitlab.com",
                 "owner": "gp-test-group",
                 "name": "gp-test-empty-project",
                 "cloneUrl": "https://gitlab.com/gp-test-group/gp-test-empty-project.git",
                 "private": false,
-                "defaultBranch": null
+                "defaultBranch": "main"
```

Comment:
this is somehow independent, but I believe they broke the API if that's really true now for uninitialized repos. how can an empty repo has a branch?